SPM Example/Secrets.xcconfig

```diff
@@ -0,0 +1,16 @@
+//
```

Comment:
This file should not be pushed to Git. We'll configure organization or repository secrets to handle these values.

Please remove this file.